components/fc.css

```diff
@@ -18,3 +18,13 @@
 .axis text { 
   font: 10px sans-serif; 
 }
+
+.gridlines { 
+  stroke: #ccc;
+  stroke-width: 1.0;
+}
+
+.chartArea rect.background {
```

Comment:
I guess whether we keep this or not depends on the future of chartLayout? Still worth merging this PR all the same